lib/plugins/physics.js

```diff
@@ -45,10 +45,14 @@ function inject (bot, { physicsEnabled, maxCatchupTicks }) {
     x: 0,
     y: 0,
     z: 0,
+    dx: 0, // 1.21.3
+    dy: 0,
+    dz: 0,
     yaw: 0,
     pitch: 0,
     onGround: false,
-    time: 0
+    time: 0,
+    flags: bot.registry.version['>=']('1.21.3') ? {} : 0
```

Comment:
Should probably be a feature

Reply:
I reverted this, I messed up the client and serverbound position packets (serverbound doesn't have dx/dy/dz and changed onGround bool to a packed flag int so we dont need a version check)